pkg/psmdb/mongo/mongo.go

```diff
@@ -170,7 +170,7 @@ func WriteConfig(ctx context.Context, client *mongo.Client, cfg RSConfig) error
 	}
 
 	if err := res.Decode(&resp); err != nil {
-		return errors.Wrap(err, "failed to decode to replSetReconfigResponse")
+		return errors.Wrap(err, "failed to decoge to replSetReconfigResponse")
```

Comment:
decode is the right one =)

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/719/commits/de7cb57eba854c2446adfdc36261fbb71ebc1b04